packages/manager/apps/dedicated/client/app/account/contacts/service/contacts-service.service.js

```diff
@@ -50,7 +52,13 @@ export default class {
         const availableServices = filter(services, (service) =>
           AVAILABLE_SERVICES.includes(service.category),
         );
-        return availableServices.map((service) => new BillingService(service));
+        const promises = [];
+        forEach(availableServices, (service) => {
+          promises.push(this.getServiceInfos(service));
+        });
```

Comment:
```suggestion
        const promises = availableServices.map((service) => this.getServiceInfos(service));
```
Also, you'll be able to remove `import forEach …`